Extension/src/LanguageServer/configurations.ts

```diff
@@ -1622,13 +1624,22 @@ export class CppProperties {
             let paths: string[] = [];
             let compilerPath: string | undefined;
             for (const pathArray of [ (currentConfiguration.browse ? currentConfiguration.browse.path : undefined),
-                currentConfiguration.includePath, currentConfiguration.macFrameworkPath, currentConfiguration.forcedInclude ]) {
+                currentConfiguration.includePath, currentConfiguration.macFrameworkPath ]) {
                 if (pathArray) {
                     for (const curPath of pathArray) {
                         paths.push(`${curPath}`);
                     }
                 }
             }
+            // Skip the relative forcedInclude files.
+            if (currentConfiguration.forcedInclude) {
+                for (const file of currentConfiguration.forcedInclude) {
+                    const resolvedFilePath: string = this.resolvePath(file, isWindows);
+                    if (util.checkFileExistsSync(resolvedFilePath) || util.checkDirectoryExistsSync(resolvedFilePath)) {
```

Comment:
You don't need to check it exists here.  Use this API instead to decide: https://nodejs.org/api/path.html#path_path_isabsolute_path